src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.targets

```diff
@@ -555,8 +562,8 @@ The .NET Foundation licenses this file to you under the MIT license.
     <ItemGroup Condition ="'$(_targetOS)' == 'wasi'" >
       <CustomLinkerArg Include="--sysroot=&quot;$(WASI_SDK_PATH.Replace(&quot;\&quot;, &quot;/&quot;))/share/wasi-sysroot&quot;" />
       <CustomLinkerArg Include="@(CustomLinkerArgExport->'-Wl,--export=%(Identity)')" />
-      <!-- Wasi has lots of undefined symbols currently, mostly globalization -->
-      <CustomLinkerArg Include="-Wl,--unresolved-symbols=ignore-all -Wl,--error-limit=0" />
+      <!-- We want to put 'warn-unresolved-symbols' here, but wasm-ld crashs on that. Oops. -->
+      <CustomLinkerArg Include="-Wl,--unresolved-symbols=ignore-all" />
```

Comment:
Will have to debug and file a bug on LLVM about this (if it hasn't been fixed already)...